src/server/connection.cpp

```diff
@@ -27,12 +28,11 @@ boost::asio::ip::tcp::socket &Connection::socket() { return TCP_socket; }
 /// Start the first asynchronous operation for the connection.
 void Connection::start()
 {
-    TCP_socket.async_read_some(
-        boost::asio::buffer(incoming_data_buffer),
-        strand.wrap(boost::bind(&Connection::handle_read,
```

Comment:
This means we no longer need to explicitly use the strand in our connection event handlers.